example/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.10)
+
+project(greentea-client-example
+    VERSION 0.1
+    DESCRIPTION "Example for the Greentea client"
+    LANGUAGES C CXX
+)
+
+add_subdirectory(.. greentea-client-build)
```

Comment:
I don't really like the `..` here. It makes the example not copy pastable into other projects as easily.

Could we instead `add_subdirectory(example)` from the root CMakeLists.txt? We can then be ensured that the `greentea-client` target will have been created and available to link with.

With this change, the instructions in the commit message would be more like:
```
cmake -S . -B cmake_build -GNinja
cmake --build cmake_build
```

Reply:
Done